R/dftype.R

```diff
@@ -0,0 +1,15 @@
+#' Explore the type of data frame variables and columns.
+#'
+#' Create a new data frame which contians the type of columns and variables for the input data frame.
+#'  Furthermore, if there are non-numeric columns, it will return the unique values of non-numeric columns and their length.
```

Comment:
Just a super minor correction (the extra space) :)

```suggestion
#' Furthermore, if there are non-numeric columns, it will return the unique values of non-numeric columns and their length.
```